opentelemetry-otlp/tests/integration_test/README.md

```diff
@@ -15,3 +15,42 @@ popped back out into the files output by the collector.
 * Docker, for the test container
 * TCP/4317 and TCP/4318 free on your local machine. If you are running another
   collector, you'll need to stop it for the tests to run.
+
+## Test Matrix
+
```

Comment:
Should we add:

```suggestion
We aim to support and provide integration test coverage for every permutation of [A/B/C]
```

... or something along these lines, so its clear what the desired state here is? It'd be helpful to identify gaps. I think optimally we'd link out to some architectural or design goal elsewhere, if we had one handy.